### ISSUE TITLE
Downloads/links, bookmarks and timespans

### ISSUE THREAD

**Issue by TimeScience** [borevitzlab/timeplayer-9000#13]
Nice work. This is great. I really like the pop up and it is great we can now bookmark, share and embed time periods within a timestream. 

From a user experience side I'm thinking though that if a person wants to just bookmark or get the URL for the whole timestream, they might think these buttons would do it but then get the specific timespan instead. This isn't a huge deal, but in terms of a clean user experience, can we think of a way to either tell the user their bookmark is just for the selected timespan or add a "url of whole timestream" and embed whole timestream" buttons as well? Is it hard to have buttons pop out the side of the popped up button? Like for the "embed" button, it could pop out to "embed code for current time range" and "embed code for timestream". something like that. let's discuss

**Comment by TimeScience** [comment]
I agree in general,  but if you think about this as maintaining consistency
in the user experience, that pop up list is about bookmarking stuff and
sharing, but then we don't have anywhere for the user to get general links
like "just share or embed this whole timestream".

**Comment by gdunstone** [comment]
There is the address bar?

**Comment by TimeScience** [comment]
the address bar does not provide an embed link

--
​
Tim Brown
Research Fellow, Center for Plant Energy Biology
Australian National University, Research School of Biology, Canberra ACT
0200
*bit.ly/Tim_ANU <http://bit.ly/Tim_ANU>* | tim.brown@anu.edu.au
​
+61 0419-006-102
​ (m) |
​
Skype: *TimeScience*
